package-lock.json

```diff
@@ -14,7 +14,7 @@
       },
       "engines": {
         "node": "^18.20.2",
-        "npm": "^10.9.0"
+        "npm": "^10.7.0"
```

Comment:
let's not commit this or also update package.json to represent the same version

Reply:
an earlier PR made the versions in package-lock and package different, this is correcting them